src/core/api/public_api.ts

```diff
@@ -1326,10 +1325,8 @@ class Player extends EventEmitter<PLAYER_EVENT_STRINGS, any> {
       } else if ((time as { wallClockTime? : number }).wallClockTime != null) {
         positionWanted = isDirectFile ?
           (time as { wallClockTime : number }).wallClockTime :
-          fromWallClockTime(
-            (time as { wallClockTime : number }).wallClockTime * 1000,
-            (manifest as Manifest) // is TS or I dumb here?
-          );
+          (time as { wallClockTime : number }).wallClockTime -
+            ((manifest as Manifest).availabilityStartTime || 0);
```

Comment:
We don't bind it to the limit of the content anymore, as that make not much sense